addon/components/bootstrap-checkbox-group.js

```diff
@@ -13,4 +13,18 @@ export default Ember.Component.extend(InputableMixin, {
   label: null,
 
   required: false,
+
+  actions: {
+    onChange(option, isChecked) {
+      const selected = this.get('selected');
```

Comment:
This should probably use `makeArray(this.get('selected'))` here to make sure that this is an array and not `null`